templates/zerver/help/add-emoji.md

```diff
@@ -11,7 +11,6 @@ built-in emoji menu, pasting Unicode, or typing the emoji code.
 
```

Comment:
I think there is a typo on the first line. It should be 'Emoji *allows* you to'.